src/views/BusinessCase/AlternativeSolution/index.tsx

```diff
@@ -0,0 +1,230 @@
+import React from 'react';
+import { Field, FormikProps } from 'formik';
+import Label from 'components/shared/Label';
+import HelpText from 'components/shared/HelpText';
+import TextField from 'components/shared/TextField';
+import TextAreaField from 'components/shared/TextAreaField';
+import FieldGroup from 'components/shared/FieldGroup';
+import FieldErrorMsg from 'components/shared/FieldErrorMsg';
+import Button from 'components/shared/Button';
+import EstimatedLifecycleCost from 'components/EstimatedLifecycleCost';
+import { defaultProposedSolution } from 'views/BusinessCase';
+import {
+  BusinessCaseModel,
+  ProposedBusinessCaseSolution
+} from 'types/businessCase';
+
+type AlternativeSolutionProps = {
+  formikProps: FormikProps<BusinessCaseModel>;
+  altLetter: string;
+  handleToggleAlternative?: () => void;
+};
+
+const AlternativeSolution = ({
+  formikProps,
+  altLetter,
+  handleToggleAlternative
+}: AlternativeSolutionProps) => {
+  const { values } = formikProps;
+  const altLabel = `Alternative ${altLetter}`;
+  const altId = `alternative${altLetter}`;
+  const altValues = ((letter: string): ProposedBusinessCaseSolution => {
+    switch (letter) {
+      case 'A':
+        return values.alternativeA;
+      case 'B':
+        return values.alternativeB || defaultProposedSolution;
+      default:
+        return defaultProposedSolution;
+    }
+  })(altLetter);
+
+  return (
+    <>
+      <h1 className="font-heading-xl">Alternatives Analysis</h1>
+      <div className="tablet:grid-col-9">
+        <div className="easi-business-case__name-wrapper">
+          <h2>{altLabel}</h2>
+          {altLetter === 'B' && (
+            <Button
+              type="button"
+              className="margin-left-2"
+              unstyled
+              onClick={handleToggleAlternative}
+            >
+              Remove Alternative B
+            </Button>
+          )}
+        </div>
+        <FieldGroup scrollElement={`${altId}.title`} error={false}>
+          <Label htmlFor={`BusinessCase-${altId}Title`}>
```

Comment:
Interpolating the `altId` is common around this view in order to reuse this view for Alternative A and B.

These values will be able to map the label to field as well as update the values in Formik.